### ISSUE TITLE
Improve "Coordinates" section of the tutorial

### ISSUE THREAD

**Issue by dstansby** [sunpy/sunpy#6864]
### Describe the feature

Things that I've noticed that could be improved at a glance are:

- Move "Using Coordinates with Maps" to the map section of the tutorial (at the point of coordinates in the tutorials we haven't introduced the concept of maps)

There might be other improvements that can be made too 🧭

**Comment by dstansby** [comment]
I'm going to de-milestone this, since we can always backports improvements to the tutorials

**Comment by wtbarnes** [comment]
I do not agree. The refactor of the documentation is our biggest improvement for 5.0. There is not any point in pushing something out the door that is incomplete.

**Comment by dstansby** [comment]
I don't think pushing out the coordinate tutorial in it's current state would be considered incomplete? I think there's incremental improvements to be made, and given they can be backported and no-one opened a PR before feature freeze that we shouldn't hold up 5.0 for it.
